packages/docker-gc/extra/dcos-docker-gc.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Docker GC: periodically garbage collects Docker containers and images
 ConditionPathExists=/opt/mesosphere/etc/docker_gc_enabled
+Before=dcos-mesos-slave.service
```

Comment:
https://github.com/dcos/dcos/blob/master/docs/systemd-rules.md

Reply:
> `Before=`, `After=` are discouraged. They are not strong guarantees

Reply:
Can we use filter to not remove dcos networks?

Reply:
:+1: while I agree in general, I would advocate that this be one of the exceptions, as it stands its a coin flip as to if docker apps can use container networks.

Reply:
@janisz I'd probably run `docker container prune`, `docker image prune`, etc, but not `docker network prune` or `docker system prune`.

Reply:
> Can we use filter to not remove dcos networks?

Not currently, as no labels are added to the network to be able to filter on

Reply:
@urbanserj @janisz I like the idea of just running the `container` and `image` `prunes`. If that's amenable to all, I'll change to that here.